yb-voyager/src/query/queryissue/constants.go

```diff
@@ -38,30 +38,38 @@ const (
 	EXPRESSION_PARTITION_WITH_PK_UK          = "EXPRESSION_PARTITION_WITH_PK_UK"
 	MULTI_COLUMN_LIST_PARTITION              = "MULTI_COLUMN_LIST_PARTITION"
 	INSUFFICIENT_COLUMNS_IN_PK_FOR_PARTITION = "INSUFFICIENT_COLUMNS_IN_PK_FOR_PARTITION"
-	XML_DATATYPE                             = "XML_DATATYPE"
-	XID_DATATYPE                             = "XID_DATATYPE"
-	GEOMETRY_DATATYPE                        = "GEOMETRY_DATATYPE"
-	GEOGRAPHY_DATATYPE                       = "GEOGRAPHY_DATATYPE"
-	BOX2D_DATATYPE                           = "BOX2D_DATATYPE"
-	BOX3D_DATATYPE                           = "BOX3D_DATATYPE"
-	TOPOGEOMETRY_DATATYPE                    = "TOPOGEOMETRY_DATATYPE"
-	RASTER_DATATYPE                          = "RASTER_DATATYPE"
-	PG_LSN_DATATYPE                          = "PG_LSN_DATATYPE"
-	TXID_SNAPSHOT_DATATYPE                   = "TXID_SNAPSHOT_DATATYPE"
-
-	POINT_DATATYPE   = "POINT_DATATYPE"
-	LINE_DATATYPE    = "LINE_DATATYPE"
-	LSEG_DATATYPE    = "LSEG_DATATYPE"
-	BOX_DATATYPE     = "BOX_DATATYPE"
-	PATH_DATATYPE    = "PATH_DATATYPE"
-	POLYGON_DATATYPE = "POLYGON_DATATYPE"
-	CIRCLE_DATATYPE  = "CIRCLE_DATATYPE"
-
-	ARRAY_OF_ENUM_DATATYPE = "ARRAY_OF_ENUM_DATATYPE"
-	USER_DEFINED_DATATYPE  = "USER_DEFINED_DATATYPE"
-	TSQUERY_DATATYPE       = "TSQUERY_DATATYPE"
-	TSVECTOR_DATATYPE      = "TSVECTOR_DATATYPE"
-	HSTORE_DATATYPE        = "HSTORE_DATATYPE"
+
+	UNSUPPORTED_DATATYPE_XML             = "UNSUPPORTED_DATATYPE_XML"
+	UNSUPPORTED_DATATYPE_XID             = "UNSUPPORTED_DATATYPE_XID"
+	UNSUPPORTED_DATATYPE_GEOMETRY        = "UNSUPPORTED_DATATYPE_GEOMETRY"
+	UNSUPPORTED_DATATYPE_GEOGRAPHY       = "UNSUPPORTED_DATATYPE_GEOGRAPHY"
+	UNSUPPORTED_DATATYPE_BOX2D           = "UNSUPPORTED_DATATYPE_BOX2D"
+	UNSUPPORTED_DATATYPE_BOX3D           = "UNSUPPORTED_DATATYPE_BOX3D"
+	UNSUPPORTED_DATATYPE_TOPOGEOMETRY    = "UNSUPPORTED_DATATYPE_TOPOGEOMETRY"
+	UNSUPPORTED_DATATYPE_RASTER          = "UNSUPPORTED_DATATYPE_RASTER"
+	UNSUPPORTED_DATATYPE_PG_LSN          = "UNSUPPORTED_DATATYPE_PG_LSN"
+	UNSUPPORTED_DATATYPE_TXID_SNAPSHOT   = "UNSUPPORTED_DATATYPE_TXID_SNAPSHOT"
+	UNSUPPORTED_DATATYPE_INT8_MULTIRANGE = "UNSUPPORTED_DATATYPE_INT8_MULTIRANGE"
+	UNSUPPORTED_DATATYPE_INT4_MULTIRANGE = "UNSUPPORTED_DATATYPE_INT4_MULTIRANGE"
+	UNSUPPORTED_DATATYPE_DATE_MULTIRANGE = "UNSUPPORTED_DATATYPE_DATE_MULTIRANGE"
+	UNSUPPORTED_DATATYPE_NUM_MULTIRANGE  = "UNSUPPORTED_DATATYPE_NUM_MULTIRANGE"
+	UNSUPPORTED_DATATYPE_TS_MULTIRANGE   = "UNSUPPORTED_DATATYPE_TS_MULTIRANGE"
+	UNSUPPORTED_DATATYPE_TSTZ_MULTIRANGE = "UNSUPPORTED_DATATYPE_TSTZ_MULTIRANGE"
```

Comment:
@ShivanshGahlot here also we should follow same format in issue type also.

Lets not separate out the datatype with `_` where actual datatype is continuous.

Reply:
Refer - https://www.postgresql.org/docs/current/rangetypes.html